Automattic/vip-go-mu-plugins
# Debug Bar 1.0.1

0.9 => 1.0.1

## Steps to Test

- Log in as a Support User
- Make sure Debug Bar loads and nothing goes 💥
